### ISSUE TITLE
Admin:Measurement Types: table headings needed

### ISSUE THREAD

**Issue by NiquiCara** [rubyforgood/abalone#560]
The table needs a scope for the table headings. Probably `scope="col"`. The column for the action buttons needs words in the heading.

![measurement-types-table-header](https://user-images.githubusercontent.com/45127691/102564005-8d0c2980-40a8-11eb-92a0-f35a5308598c.PNG)

**Comment by hairedfox** [comment]
@NiquiCara Did you mean that the table needs the border and add the heading of 2 actions should be "Action"? Please confirm. And I'd love to get this issue done.

**Comment by CraigJZ** [comment]
Hi @hairedfox Yes, what you proposed should resolve this issue. Happy to assign you if you would like to take it on. Thanks!
